arangod/IResearch/IResearchView.cpp

```diff
@@ -154,36 +149,33 @@ struct IResearchView::ViewFactory : public arangodb::ViewFactory {
     auto links = properties.hasKey(StaticStrings::LinksField)
                      ? properties.get(StaticStrings::LinksField)
                      : velocypack::Slice::emptyObjectSlice();
-    auto res = engine.inRecovery()
-                   ? Result()  // do not validate if in recovery
-                   : IResearchLinkHelper::validateLinks(vocbase, links);
+    auto r = engine.inRecovery()
```

Comment:
NIT: Please don`t use single letter variables with lifespan longer than several lines. This makes reading hard.